supermercado/scripts/cli.py

```diff
@@ -42,22 +45,28 @@ def union(inputtiles, parsenames):
     for u in unioned:
         click.echo(json.dumps(u))
 
-cli.add_command(union)
-
 
-@click.command('burn')
+@cli.command('burn')
 @cligj.features_in_arg
 @cligj.sequence_opt
-@click.argument('zoom', type=int)
+@click.argument(
+    'zoom',
+    type=str
+)
 def burn(features, sequence, zoom):
     """
     Burn a stream of GeoJSONs into a output stream of the tiles they intersect for a given zoom.
     """
     features = [f for f in super_utils.filter_polygons(features)]
 
-    tiles = burntiles.burn(features, zoom)
-    for t in tiles:
-        click.echo(t.tolist())
+    # Resolve the minimum and maximum zoom levels for export.
+    zooms = list(map(int, zoom.split('..')))
+    minzoom = zooms[0]
+    maxzoom = zooms[0] if len(zooms) == 1 else zooms[1]
 
+    tiles = []
+    for zoom in range(minzoom, maxzoom + 1):
+        tiles.append(burntiles.burn(features, zoom))
 
-cli.add_command(burn)
\ No newline at end of file
+    for t in list(itertools.chain.from_iterable(tiles)):
```

Comment:
I don't think we need to make this a list before iterating over all the output tiles.